odxtools/odxtypes.py

```diff
@@ -241,3 +241,18 @@ def isinstance(self, value: Any) -> bool:
             return True
         else:
             return False
+
+    def __str__(self) -> str:
+        if self == DataType.A_INT32:
+            return "int"
+        elif self == DataType.A_UINT32:
+            return "uint"
+        elif self in (DataType.A_FLOAT32, DataType.A_FLOAT64):
+            return "float"
+        elif self == DataType.A_BYTEFIELD:
+            return "bytefield"
+        elif self in (DataType.A_UNICODE2STRING, DataType.A_ASCIISTRING, DataType.A_UTF8STRING):
+            return "string"
+        else:
+            odxraise(f"Type info for type {self.value}", NotImplementedError)
```

Comment:
Avoid raising in `__str__`, maybe just return `f"<unknown type: {self.value}>"`

Reply:
good point. fixed: https://github.com/mercedes-benz/odxtools/pull/348/commits/4a9f43a21510ada21c4b8a3ad55bc268ec871fa9